src/main/java/org/example/hw7/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,44 @@
+package org.example.hw7;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class CoffeeOrderBoard {
+
+    private Set<Order> orders;
+
+    public CoffeeOrderBoard() {
+        this.orders = new HashSet<>();
+    }
+
+    public void add(Order order) {
+        orders.add(order);
+    }
+
+    public Order deliver() {
```

Comment:
set is a unique collection that stores data in an unordered manner.  deliver() need to return the orders in proper order.